lib/ts/axios.ts

```diff
@@ -243,7 +243,9 @@ export function responseInterceptor(axiosInstance: any) {
                 !doNotDoInterception &&
                 // we do not call doesSessionExist here cause the user might override that
                 // function here and then it may break the logic of our original implementation.
-                !((await getLocalSessionState(true)).status === "EXISTS")
+
+                // Calling getLocalSessionState with tryRefresh: false, since the session would have been refreshed in the try block if expired.
+                (await getLocalSessionState(false)).status === "NOT_EXISTS"
             ) {
```

Comment:
should we get rid of this if block entirely? Cause if the session doesn't exist, it means the front token and anti csrf token should already be removed?

Reply:
We do not know for sure if the anti-csrf one is removed or not, but the front-token shouldn't exist here anyway.